data/presets/fields/surveillance/type.json

```diff
@@ -0,0 +1,8 @@
+{
+    "key": "surveillance:type",
```

Comment:
Although I default to `camera` as well when I map and I don't view the other variants meaningful either, however it has become kind of de facto mandatory.

For example, this visualization also requires many fields to work (described on right panel):
https://kamba4.crux.uberspace.de/

Although, I'll probably add `camera:direction` to the form as well because I find it easy to add in practice.